.github/workflows/package-size.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '18'
-      - run: yarn
+      - run: yarn --ignore-engines
```

Comment:
I the PR description you've said:

> It fails on Node 16 otherwise because the new eslint dependencies don't support it.

But this ci job is running only with `node-version: '18'` when is this job failing?

Reply:
On the main branch yes, but on v4.x it runs on Node 16. I guess there is no reason not to use Node 18 everywhere for this specifically, so I could change the version instead of that makes more sense, although not sure how since it's already changed in a larger commit that is now landed.

Reply:
I guess I could bump it further to for example 20+